compiler/plc_diagnostics/src/error_codes/E093.md

```diff
@@ -0,0 +1,22 @@
+# Conditional Loop Values
+
+Any conditional value used in a for-loop, i.e. the counter, start, end and step-value must be of the same numerical
```

Comment:
I couldn't think of a generic name for the `counter`, `start`, `end` and `step` values used in a for-loop so I just used "conditional values" here which is kind of misleading for everything but `start`. Is this still OK or does anyone have a better name suggestion here?

Reply:
you can call the last also initial, increment, and limit...
i would not say this is better... its just different... so i am also fine with start, step and end...